api_tests/package.json

```diff
@@ -52,18 +30,39 @@
         "@types/scrypt": "^6.0.0",
```

Comment:
Wait I am confused, did you remove the `devDependencies` section??

Reply:
Check the first commit here: https://github.com/comit-network/comit-rs/pull/1672/commits/5d3c57d360e7de605f994082a9ddbdb7438527e4

Reply:
typo in commit: `distuingish` -> `distinguish`